source/includes/scripting/fact-require-ems-modules.rst

```diff
@@ -0,0 +1,12 @@
+Node.js modules are packaged in two formats. Newer modules use the
+``ECMAScript`` standard. Older modules use the ``CommonJS`` (CJS)
```

Comment:
[optional suggestion]

I know it's only two items, but you might consider bulleting these out for increased scannability.

- Newer modules use the ``ECMAScript`` standard.
- Older modules use the ``CommonJS`` (CJS) format.